artman/config/loader.py

```diff
@@ -24,6 +24,14 @@
 from google.protobuf import json_format
 
 from artman.config.proto.config_pb2 import Artifact, Config
+from artman.utils.logger import logger
+
+# Error messages
+CONFIG_NOT_FOUND_ERROR_MESSAGE_FORMAT = (
+    'Artman YAML cannot be found at `%s`. Please check the file location.')
+# TODO(ethanbao): Add a reference link once Artman schema spec is publicly
+# available.
+INVALID_CONFIG_ERROR_MESSAGE_FORMAT = 'Artman YAML %s is invalid.'
```

Comment:
Is there no way to get an error message or a line number out of the parser?

Reply:
We lose the track in the mid. Because protobuf python library only support validation of a json string not yaml string, I ended up converting yaml string to json string first, the line number mapping is not retained.

If the yaml file itself is in a invalid format (without checking it against a protobuf message type schema), the line number is returned as part of yaml.parser.ParserError,

Reply:
I see. I hope the `json_format.ParseError` gives enough information to the user to debug if the yaml is valid but the instance of the proto is not.